docs/walk-through/artifacts.md

```diff
@@ -257,3 +257,12 @@ kubectl patch workflow my-wf \
     --type json \
     --patch='[ { "op": "remove", "path": "/metadata/finalizers" } ]'
 ```
+
+A flag has been added to the Workflow Spec called `forceFinalizerRemoval` (see [here](../fields.md#workflowlevelartifactgc)) to force the finalizer's removal even if Artifact GC fails:
+```yaml
+spec:
+  artifactGC:
+    strategy: OnWorkflowDeletion 
+    forceFinalizerRemoval: true
+
+```
```

Comment:
Should I indicate a release version for this? (unfortunately, it hasn't been released so I could only guess) Otherwise, users should assume documentation is essentially guaranteed to be correct for "latest".

Reply:
this will be confusing until the feature is released. maybe we could hold on merging or say "coming in v3.5"?